src/App/Fossa/Main.hs

```diff
@@ -149,6 +151,10 @@ appMain = do
 dieOnWindows :: String -> IO ()
 dieOnWindows op = when (SysInfo.os == windowsOsName) $ die $ "Operation is not supported on Windows: " <> op
 
+
+parseCommaSeparatedFileArg :: Text -> IO [Path Abs File]
+parseCommaSeparatedFileArg arg = sequence $ validateFile . T.unpack <$> T.splitOn arg ","
```

Comment:
[nit]: The mix of operators is confusing (to me at least).  I'm sure it's doing what it looks like, from right to left, but it's a little hard when `$`, `.`, and `<$>` are all mixed in a one-liner.  I'd recommend parens for clarity:
```suggestion
parseCommaSeparatedFileArg arg = sequence (validateFile . T.unpack <$> T.splitOn arg ",")
```